src/PKSim.Core/Mappers/ExpressionProfileToExpressionProfileBuildingBlockMapper.cs

```diff
@@ -25,7 +25,11 @@ public class ExpressionProfileToExpressionProfileBuildingBlockMapper : PathAndVa
 
       protected override IFormula TemplateFormulaFor(IParameter parameter, IFormulaCache formulaCache, ExpressionProfile expressionProfile)
       {
-         //for expression profile, all formula are in the calculation method EXPRESSION_PARAMETERS
+         var formula = parameter.Formula;
+         //for expression profile, all formula are in the calculation method EXPRESSION_PARAMETERS unless they are distributed table
+         if (formula is TableFormula tableFormula)
```

Comment:
we have two parameters with a table formula. I am returning it as is for now. Wonder if this is a problem?

Reply:
Don't see a problem. (Does not mean there isn't one, though 😆)